README.md

```diff
@@ -46,30 +48,17 @@ nextflow run main.nf
     --newgenome <new_genome.fa> \
     --vcffile <source.vcf> \
     --outfile <remap.vcf> \
-    [--flankingseq 50] \
-    [--scorecutoff 0.6] \
-    [--diffcutoff 0.04]
```

Comment:
After removing these lines, I think the final `\` on the previous line should be removed as well

Reply:
Done